release_docs/RELEASE.txt

```diff
@@ -314,6 +314,24 @@ Bug Fixes since HDF5-1.10.9 release
 
       (ADB - 2023/02/16 GH-1546,GH-2259)
 
+    - Remove Javadoc generation
+
+      The use of doxygen now supersedes the requirement to build javadocs. We do not
+      have the resources to continue to support two documentation methods and have
+      chosen doxygen as our standard.
+
+      (ADB - 2022/12/19)
```

Comment:
These dates are from develop.

Reply:
Probably ok since they are before this release and after the 1.10.9 release?